### PR TITLE
[RET-212][RET-213][RET-214] Sign in Page

### PR DESCRIPTION
**Pull request by jcmagsay**
### What's this PR do?
- Abstract Page Hero
- Create new Layout for Page Hero Actions

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-212
https://mondorobot.atlassian.net/browse/RET-213
https://mondorobot.atlassian.net/browse/RET-214

### Screenshots (if appropriate)
screen shot 2017-04-20 at 5 09 46 pm

### Who should review this PR?
@jcmagsay @mhfen

### CODE REVIEW

File: src/assets/styles/base/elements.scss [hunk 1]
```diff
@@ -44,3 +44,9 @@ a {
 button {
   cursor: pointer;
 }
+
+input:-webkit-autofill,
+textarea:-webkit-autofill,
+select:-webkit-autofill {
+  background-color: $COLOR_TRANSPARENT !important;
+}
```
Comment (mhfen) [inline src/assets/styles/base/elements.scss]: Is this meant to hide auto-completed forms? Using `autocomplete="off"` is cross-browser standard.

File: src/assets/styles/helpers/functions.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* --------------------------
+ MIXIN FUNCTION DECLARATIONS
+----------------------------- */
+/*
+ Returns a multiple of the base rhythm unit of the design
+ Converts to rems
+*/
+@function rhythm($multiplier) {
+  @return #{$multiplier * $RHYTHM_BASE / 10}rem;
+}
```
Comment (mhfen) [inline src/assets/styles/helpers/functions.scss]: Cool, glad you found this. Helped Steve and I out a lot.

File: src/assets/styles/modules/objects/set/set.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+.set-bd_mega {
+  margin-top: rhythm(5);
+}
+
+.set-bd_isolated {
+  margin-top: rhythm(10);
+}
+
+.set-ft {
```
Comment (mhfen) [inline src/assets/styles/modules/objects/set/set.scss]: Sweet, nice work on the vertical rhythm in this object. Steve gave me some feedback that using `-bd` and `-ft` redundantly doesn't add anything. Dropping all of the `-ft` children and instead just calling `-bd` children `set-item` makes them more abstract. Feel like trying that out?

File: src/assets/styles/base/elements.scss (hunk 1 above)
Comment (mhfen) [inline src/assets/styles/base/elements.scss]: And I just saw you did that below :)

File: src/assets/styles/modules/objects/set/set.scss (hunk 3 above)
Comment (jcmagsay) [inline src/assets/styles/modules/objects/set/set.scss]: Definitely. I was feeling the same way when writing the code, just didn't want to break Mondo standards

**Pull request merged**